src/proto.rs

```diff
@@ -652,43 +743,22 @@ pub struct AuthenticatorData {
     /// The optional attestation.
     pub(crate) acd: Option<AttestedCredentialData>,
     /// Extensions supplied by the device.
-    pub(crate) extensions: Option<serde_cbor::Value>,
-}
-
-impl AuthenticatorData {
```

Comment:
I don't think we need this anymore, but wasn't sure what it's original motivation was